src/Smoker/SmokeTest.cs

```diff
@@ -92,9 +92,11 @@ public async Task<bool> Run()
                         // validate the response
                         if (resp.StatusCode == System.Net.HttpStatusCode.OK && r.Validation != null)
                         {
+                            res = ValidateStatusCode(r, resp);
                             res = ValidateContentType(r, resp);
                             res += ValidateContentLength(r, resp);
```

Comment:
Should the 2nd line be: res *+=* ValidateContentType.. ? (Same for other validate blocks below)

Reply:
I added a ValidateAll() to ensure consistency across run once, run from web and run loop